doc/nrf/app_build_system.rst

```diff
@@ -156,6 +156,43 @@ The following scenarios are possible:
 
 .. output_build_files_table_end
 
+.. _app_build_mcuboot_output:
+
+MCUboot output build files
+==========================
+
++------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| File                                                             | Description                                                                                                                                                                                  |
++==================================================================+==============================================================================================================================================================================================+
+| :file:`dfu_application.zip`                                      | Zip file containing both the MCUboot-compatible update image for one or more cores and a manifest describing its contents (:file:`app_update.bin` and a :file:`manifest.json` file).         |
++------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| :file:`app_update.bin`                                           | Signed variant of the firmware in binary format (as opposed to intelhex).                                                                                                                    |
+|                                                                  | This file can be uploaded to a server as FOTA image.                                                                                                                                         |
++------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
Looks like `dfu_mcuboot.zip` is missing from the list. Here is a suggested description based on the discussion in the jira-task, but feel free to modify.

```suggestion
+------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| :file:`dfu_mcuboot.zip`                                           | Zip file containing two versions of MCUBoot linked against different address spaces corresponding to slot0 (s0) and slot1 (s1) and a manifest JSON file describing their MCUBoot version number(MCUBOOT_IMAGE_VERSION), NSIB version number(FW_INFO), board type, etc.                                                                                                                    |
|                                                                  | This file can be used by FOTA servers to serve both s0 and s1 to the device, and the device can select the firmware file for the slot that is currently not in use .                                                                                                                                         |
+------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Reply:
added